include/librealsense2/h/rs_context.h

```diff
@@ -63,6 +63,13 @@ rs2_device* rs2_context_add_device(rs2_context* ctx, const char* file, rs2_error
  */
 void rs2_context_remove_device(rs2_context* ctx, const char* file, rs2_error** error);
 
+/**
+ * Removes tracking module.
+ * query_devices locks the tracking module. 
```

Comment:
The description of both the original issue and the solution is not clear - please review and adjust.